mapcompose/src/main/java/ovh/plrapps/mapcompose/ui/state/ZoomPanRotateState.kt

```diff
@@ -48,7 +48,10 @@ internal class ZoomPanRotateState(
             recalculateMinScale()
         }
 
+    internal val areGesturesEnabled get() = isRotationEnabled || isScrollingEnabled || isZoomingEnabled
     internal var isRotationEnabled = false
+    internal var isScrollingEnabled = true
+    internal var isZoomingEnabled = true
 
```

Comment:
I know that this is how its was done before with `isRotationEnabled`, but I'm suspecting that these 3 flags (`isZoomingEnabled`, `isScrollingEnabled`, and `isRotationEnabled`) should delegate their value to a State. Like: `internal var isZoomingEnabled by mutableStateOf(true)`.
I'm suspecting that if we disable all gestures, and if we dynamically enable gestures (after a while), no new composition is happening (since these flags aren't states) when the user scrolls or zoom. Can you confirm?

Reply:
That's excellent, thank you!